iceoryx2-bb/elementary/src/cyclic_tagger.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2025 Contributors to the Eclipse Foundation
```

Comment:
Renamed file, was known as `visitor.rs`. All classes are renamed to `CyclicTagger**`